src/natcap/invest/wave_energy.py

```diff
@@ -101,13 +101,13 @@
                 "A GDAL-supported raster file containing a digital elevation "
                 "model dataset that has elevation values in meters.  Used to "
                 "get the cable distance for wave energy transmission."),
-            "name": "Global Digital Elevation Model"
+            "name": "digital elevation model"
```

Comment:
Is this required to be global?  The sample dataset certainly is, and if it _is_ assumed/required to be global, that's an important bit of information to make VERY clear.

Reply:
The DEM only needs to cover every point in the wave points vector, so it technically doesn't have to be global in all cases. But since the model uses particular wave point vectors (West Coast, East Coast, North Sea 4m, North Sea 10m, Australia, Global) maybe it's best to point people to the sample global DEM, that's known to cover all the points?

Reply:
That is a slightly more nuanced requirement than can easily be expressed in a label ... no wonder it refers to the global DEM, since that'll work for all cases.

Let's leave off the global note.  This will make it match what's described in the UG chapter, and better match what's required anyways.

Reply:
Oh you're right, the UG does talk about the possibility of using different DEMs. Will leave out the "global".